src/map.cpp

```diff
@@ -6910,6 +6910,112 @@ void map::restock_fruits( const tripoint &p, int time_since_last_actualize )
     }
 }
 
+void map::produce_sap( const tripoint &p, int time_since_last_actualize )
+{
+    if( time_since_last_actualize <= 0 ) {
+        return;
+    }
+
+    if( ter_at( p ).id != "t_tree_maple_tapped" ) {
```

Comment:
@codemime mentioned that I should not have added t_tree_maple_tapped in mapdata.cpp which is why it was removed and the test changed there. I would appreciate if he could help us clear this issue.